cypress/utils/dashboards/commands.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import './query_enhancement/commands';
```

Comment:
we should have global commands if needed. and then query enhancmeents and dashboards can take from it vs another plugin command